README.md

```diff
@@ -1,7 +1,7 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
-Portfolio
+Trail Portfolio Project
```

Comment:
Hi @Tshepo45👌 ,Indeed you did an exceptional job but there are a few things that need to be addressed to go to next level.
I think it would be better to write more descriptive project name by writing about what the project is intended to accomplish

Reply:
**[OPTIONAL]** This is a great review comment, however, be careful because the README doesn't interact with the webpage! I think that this would be better placed in your HTML title. 🤓